MangoPay/PayOut.php

```diff
@@ -7,6 +7,20 @@
  */
 class PayOut extends Transaction
 {
+    /**
+     * The unique identifier of the bank account.
+     *
+     * @var string
+     */
+    public $BankAccountId;
+
+    /**
+     * Custom description to appear on the user’s bank statement along with the platform name.
+     * The recommended length is 12 characters – strings longer than this may be truncated depending on the bank.
+     * @var string
+     */
+    public $BankWireRef;
```

Comment:
What is the reason for adding this to the PayOut class?

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.